indico/modules/events/papers/blueprint.py

```diff
@@ -35,6 +35,12 @@
 _bp.add_url_rule('/manage/papers/enable/<any(content,layout):reviewing_type>', 'switch',
                  management.RHSwitchReviewingType, methods=('PUT', 'DELETE'))
 
+# CFP scheduling
+_bp.add_url_rule('/manage/papers/schedule', 'schedule_papers_call', management.RHScheduleCFP,
```

Comment:
I would go for `schedule_cfp`, `open_cfp`, and `close_cfp`. Papers call doesn't sound right to me.